packages/valory/skills/abstract_round_abci/tests/test_common.py

```diff
@@ -238,21 +238,6 @@ def test_get_randomness_from_api(
                     "`get_randomness_from_api`'s generator should have been exhausted."
                 )
 
-    def test_async_act_could_not_generate_from_chain(self) -> None:
-        """Test `async_act` when we cannot generate randomness from chain."""
-        # create a dummy `is_retries_exceeded` for `MagicMock`ed `randomness_api`
-        self.behaviour.context.randomness_api.is_retries_exceeded = lambda: True
-        gen = self.behaviour.async_act()
-
-        # test when `failsafe_randomness` returns `None`
-        with mock.patch.object(
-            self.behaviour,
-            "failsafe_randomness",
-            dummy_generator(None),
-        ):
-            next(gen)
-            last_iteration(gen)
-
```

Comment:
nice to link the code that covers this to facilitate the review process:

https://github.com/valory-xyz/open-autonomy/blob/c0376fecfc2035ed3e48d141777d0419fe198232/packages/valory/skills/abstract_round_abci/tests/test_common.py#L257-L267